api/proposal/proposal.go

```diff
@@ -67,7 +80,24 @@ func groupByProperty[T any, K comparable](items []T, getProperty func(T) K) map[
 	return grouped
 }
 
-func getAllProposalHandler(res http.ResponseWriter, req *http.Request) {
+// @Summary		Get all proposals
+// @Description	Get all proposals
+// @Tags			proposal
+// @Produce		json
+// @Success		200	{object}	[]DocumentProposal
+// @Failure		400	{object}	httputil.ApiError
+// @Router			/proposals [get]
+func GetAllProposalsHandler(res http.ResponseWriter, req *http.Request) {
+	if req.Method != http.MethodGet {
+		httputil.WriteError(res, http.StatusMethodNotAllowed, "invalid method")
+		return
+	}
+
+	if !middleware.GetAdmin(req) {
+		httputil.WriteError(res, http.StatusForbidden, "you are not admin")
+		return
+	}
+
 	db := util.GetDb()
 	var questions []Proposal
 	if err := db.Where(Proposal{}).Find(&questions).Error; err != nil {
```

Comment:
Why there's a `Where` without any parameter? This is not your code but can you fix it anyway?